recipe/meta.yaml

```diff
@@ -25,15 +27,31 @@ requirements:
     - wheel
   run:
     - python >=3.8
+    - astropy
+    - fiona
+    - gdal >=3.3
     - numpy
-    - gdal >=3.2.0
+    - pysheds
+    - rasterio
+    - scikit-fuzzy
+    - scikit-image
+    - shapely
 
 test:
   imports:
     - asf_tools
     - asf_tools.composite
+    - asf_tools.hand
```

Comment:
include `dem` here?
```suggestion
    - asf_tools.dem
```

Reply:
Oh yeah, missed it! Probably going to do `dem` above `hand` too keep them alphabetical